crates/protogen/proto/metastore/options.proto

```diff
@@ -229,6 +230,12 @@ message TableOptionsClickhouse {
   string table = 2;
 }
 
+message TableOptionsCassandra {
+  string connection_string = 1;
```

Comment:
I'd prefer `host` here since this isn't really a connection string, but a host name/ip to a known node.